.github/scripts/test_torch_version_torchao_version_compatibility.sh

```diff
@@ -0,0 +1,95 @@
+# Testing compatibility
+# We know that torchao .so files built using PyTorch 2.8.0 are not ABI compatible with PyTorch 2.9+. (see #2919)
+# If the version of torch is not compatible with the version of torchao,
+# we expect to skip loading the .so files and a warning should be logged but no error
+
+# Function to check torchao import with configurable expectations
+check_torchao_import() {
+    local expect_warning="$1"
+    local warning_text="$2"
+    local torch_incompatible="${3:-}"
+
+    if [ -n "$torch_incompatible" ]; then
+        output=$(TORCH_INCOMPATIBLE=1 python -c "import torchao" 2>&1)
+    else
+        output=$(python -c "import torchao" 2>&1)
+    fi
+    exit_code=$?
+
+    if [ $exit_code -ne 0 ]; then
+        echo "ERROR: Failed to import torchao"
+        echo "Output: $output"
+        exit 1
+    fi
+
+    warning_found=false
+    if [ -n "$warning_text" ] && echo "$output" | grep -i "$warning_text" > /dev/null; then
+        echo "Output: $output"
+        warning_found=true
+    fi
+
+    if [ "$expect_warning" != "$warning_found" ]; then
+        echo echo "FAILURE: expect_warning is $expect_warning but warning_found is $warning_found with message $output"
+        exit 1
+    fi
+}
+
+## torch == 2.8.0, torchao == 0.13.0
+# Uninstall torch
+pip uninstall torch
+# Uninstall torchao
+pip uninstall torchao
+# Install current compatible version of torch (2.8.0)
+pip install torch==2.8.0
```

Comment:
can we extract the versions out in variables and put them at the top of the script? That would make it easy for the person doing the torchao release to change them when the version increments